src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -57,13 +57,13 @@ export default class Mutations extends React.Component<
 
     public render() {
         // use routing if available, if not fall back to the observable variable
-        const activeTabId = this.props.routing
```

Comment:
can't find where do we use routing here, so remove routing and use gene symbol instead

Reply:
@inodb  We probably want to save the selected transcript id in the url? so we can also save selected gene.   lets discuss in a chat

Reply:
yes agreed